instance/models/openedx_instance.py

```diff
@@ -374,7 +374,6 @@ def spawn_appserver(self,
             app_server.make_active()
 
         appserver_spawned.send(sender=self.__class__, instance=self, appserver=app_server)
-
```

Comment:
@Kelketek, nit: unnecessary change. If there isn't a reason for changing this, can you revert it?